test/chain-adapters/Arbitrum_Adapter.ts

```diff
@@ -76,12 +76,13 @@ describe("Arbitrum Chain Adapter", function () {
     // Create an action that will send an L1->L2 tokens transfer and bundle. For this, create a relayer repayment bundle
     // and check that at it's finalization the L2 bridge contracts are called as expected.
     const { leafs, tree, tokensSendToL2 } = await constructSingleChainTree(dai, 1, arbitrumChainId);
-    await hubPool.connect(dataWorker).initiateRelayerRefund([3117], 1, tree.getHexRoot(), consts.mockTreeRoot);
```

Comment:
there was a merge issue from the previous PR. this is unrelated to the main change.